### DISCUSSION TITLE
Does Avalonia XPF run without internet connection (offline)

### DISCUSSION THREAD

**Discussion by samsunlu55atakum** [AvaloniaUI/Avalonia#12660] [General]
Does Avalonia XPF run without internet connection (offline)?

**Comment by maxkatz6** [comment]
Yes. XPF doesn't require internet connection in a complete and built application. You still need it for nuget and development, as per usual with any library.

**Reply by samsunlu55atakum** [reply]
Thanks a lot. How can I try avalonia xpf for free? I sent request on the website. I assumed that I am able to try it soon.

**Reply by MikeCodesDotNET** [reply]
Someone will be in touch to arrange a call.

**Reply by YuriyNev** [reply]
Sorry for being off topic. Is a call a prerequisite for getting a trial version? My spoken English leaves much to be desired.

**Reply by timunie** [reply]
I guess a chat in combination with seeing the screen might be enough if you feel not confident. Or leave a comment in the mail which language you prefer. Probably someone of the team speaks your language, but can't promise :-)
